<ui version="4.0" >
 <class>AddressClass</class>
 <widget class="QDialog" name="AddressClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Индекс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5" >
    <widget class="QLineEdit" name="index" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Регион</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="8" >
    <widget class="QLineEdit" name="region" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Район</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="8" >
    <widget class="QLineEdit" name="subregion" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Город</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="8" >
    <widget class="QLineEdit" name="city" />
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Населенный пункт</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="5" >
    <widget class="QLineEdit" name="subcity" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Улица</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="8" >
    <widget class="QLineEdit" name="street" />
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QComboBox" name="cmb_house" >
     <item>
      <property name="text" >
       <string>дом</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>владение</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="2" colspan="3" >
    <widget class="QLineEdit" name="house" />
   </item>
   <item row="6" column="5" >
    <widget class="QComboBox" name="cmb_building" >
     <item>
      <property name="text" >
       <string>корпус</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>строение</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="6" >
    <widget class="QLineEdit" name="building" />
   </item>
   <item row="6" column="7" >
    <widget class="QComboBox" name="cmb_flat" >
     <item>
      <property name="text" >
       <string>кв.</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>оф.</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="8" colspan="2" >
    <widget class="QLineEdit" name="flat" />
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Представление</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="7" >
    <widget class="QTextEdit" name="fullAddress" />
   </item>
   <item row="8" column="0" colspan="3" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Комментарий</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="7" >
    <widget class="QLineEdit" name="comment" />
   </item>
   <item row="9" column="0" colspan="10" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="9" >
    <widget class="QPushButton" name="btn_region" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9" >
    <widget class="QPushButton" name="btn_subregion" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9" >
    <widget class="QPushButton" name="btn_city" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9" >
    <widget class="QPushButton" name="btn_subcity" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="9" >
    <widget class="QPushButton" name="btn_street" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddressClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddressClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
